lib/src/erc20/systems/approval/erc20_approve.cairo

```diff
@@ -0,0 +1,11 @@
+#[system]
```

Comment:
we only need the approve system. increase / decrease can be implemented in the base and use it